projects/laji/src/app/+taxonomy/taxon/info-card/taxon-biology/taxon-biology.component.html

```diff
@@ -1,83 +1,55 @@
-                  <div style="float:left; width: 100%">
-                    <h4 *ngIf="variable.variable !== 'MX.ingressText' && !groupTranslationChecklist[activeDescription].groups[i].values[y]">
-                      {{ variable.title | multiLang }}
-                    </h4>
-                    <strong *ngIf="variable.variable === 'MX.ingressText' && !groupTranslationChecklist[activeDescription].groups[i].values[y]" [innerHTML]="variable.content | multiLang"></strong>
-                    <span *ngIf="variable.variable !== 'MX.ingressText' && !groupTranslationChecklist[activeDescription].groups[i].values[y]" [innerHTML]="variable.content | multiLang"></span>
-                    <div *ngIf="group.group === 'MX.SDVG5' && variable.variable === 'MX.habitat'">
-                      <div class="col-lg-6 col-md-6 col-xs-10">
-                        <span *ngIf="!variable.variable"><h4>{{ 'iucn.filter.habitat' | translate }}</h4></span>
-                        <laji-taxon-info-row *ngIf="taxon.primaryHabitat" [label]="'taxonomy.habitat.primary' | translate">
-                          <laji-habitat [habitat]="taxon.primaryHabitat"></laji-habitat>
-                        </laji-taxon-info-row>
-                        <laji-taxon-info-row *ngIf="taxon.secondaryHabitats" [label]="'taxonomy.habitat.secondary' | translate">
-                          <laji-habitat [habitat]="taxon.secondaryHabitats"></laji-habitat>
-                        </laji-taxon-info-row>
-                      </div>
-                      <div class="col-lg-6 col-md-6 col-xs-2" *ngIf="taxon.primaryHabitat || taxon.secondaryHabitats">
-                        <laji-info><span [innerHtml]="'taxonomy.ymparisto' | translate"></span></laji-info>
-                      </div>
-                    </div>
-                    <div *ngIf="group.group === 'MX.SDVG1'">
-                      <laji-taxon-ylesta-fields [info]="ylesta"></laji-taxon-ylesta-fields>
-                    </div>
-                  </div>
+  <div class="col-xs-12" *ngIf="taxonDescription.length > 0">
+    <select class="form-control mb-3" *ngIf="taxonDescription.length > 1" (change)="contextChange.emit(toHtmlSelectElement($event.target).value)">
+      <option *ngFor="let description of taxonDescription; let i = index" [value]="description.id" [selected]="activeDescription === i">
+        {{ description.title ? description.title : description.id }}
+      </option>
+    </select>
+    <div class="panel-group" role="tablist" aria-multiselectable="true" *ngIf="taxonDescription[activeDescription]">
+      <div *ngFor="let group of taxonDescription[activeDescription].groups; let i = index">
+        <div
+          class="panel panel-default"
+          [title]="group.title"
+          [open]="true"
+          [autoToggle]="true"
+          [index]="i"
+          laji-panel
+        >
+          <div *ngFor="let variable of group.variables; let y = index">
+            <div style="float:left; width: 100%">
```

Comment:
this styling with floats and also bs3 columns and such is not preferable, but it's okay since I guess this was not changed in this PR